Dockerfile

```diff
@@ -9,5 +9,5 @@ RUN useradd cluster-image-registry-operator
 USER cluster-image-registry-operator
 
 COPY --from=0 /go/src/github.com/openshift/cluster-image-registry-operator /usr/bin
-COPY deploy/image-references deploy/crd.yaml deploy/namespace.yaml deploy/operator.yaml deploy/rbac.yaml /manifests/
+COPY deploy/image-references deploy/00crd.yaml deploy/01namespace.yaml deploy/03operator.yaml deploy/02rbac.yaml /manifests/
```

Comment:
can we make these 00-crd.yaml, etc for better readability?

Reply:
https://github.com/openshift/origin/blob/256f7eaa7c5a718b95f4a2c1384a540eda67f8d6/pkg/oc/cli/admin/release/new.go#L488

is looking for `<number>_<whatever>` to decide if `99_<operator-name>_` should be prepended to filename. So i think `00-crd.yaml` should be fine.

Reply:
done